python/dgl/graphbolt/impl/basic_feature_store.py

```diff
@@ -57,6 +57,31 @@ def read(
         """
         return self._features[(domain, type_name, feature_name)].read(ids)
 
+    def size(
+        self,
+        domain: str = None,
+        type_name: str = None,
+        feature_name: str = None,
+    ):
+        """Get the size of the feature.
+        Parameters
+        ----------
+        domain : str
+            The domain of the feature such as "node", "edge" or "graph".
+        type_name : str
+            The node or edge type name.
+        feature_name : str
+            The feature name.
+        Returns
+        -------
+        int
+            The size of the feature.
+        """
+        if domain is None and type_name is None and feature_name is None:
+            for value in self._features.values():
```

Comment:
We don't need to return the number of different features in feature store. The key should not be optional.